modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators.Tests/Extensions.cs

```diff
@@ -9,4 +13,27 @@ public static MetadataReference CreateMetadataReference(this Assembly assembly)
     {
         return MetadataReference.CreateFromFile(assembly.Location);
     }
+
+    public static void AddGlobalConfig(this SolutionState state, Dictionary<string, string> config, bool shouldAppend = false)
```

Comment:
Probably not a need, it is false when we call it at the creation of the test state. But it could very well not be since the file should not exist at that point.